.kokoro

```diff
@@ -55,6 +55,12 @@ popd
 
 fix_bazel_imports
 
+if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+  bazel version
+  use_bazel.sh 0.20.0
```

Comment:
Tip: Make this a variable next to `KOKORO_RUNNER_VERSION`?

Reply:
Done.

Reply:
Looks like this is still inlined.

Reply:
Ah I may have forgotten to push the branch. Should be correct now.